Doc/library/textwrap.rst

```diff
@@ -17,31 +17,63 @@ If you're just wrapping or filling one or two text strings, the convenience
 functions should be good enough; otherwise, you should use an instance of
 :class:`TextWrapper` for efficiency.
 
-.. function:: wrap(text, width=70, **kwargs)
+.. function:: wrap(text, width=70, initial_indent="", subsequent_indent="", \
```

Comment:
All parameters except *text* and *width* are keyword-only.

Reply:
Thank you, this is fixed. See d6b5fe08b70102b63c02a0861cb65587aeac6144